### PR TITLE
added ability to align grid items at the bottom

### PR DESCRIPTION
**Pull request by HannaLetska**

### CODE REVIEW

File: src/elements/Grid/Grid.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ const renderGridItems = (items, widths, stacked, gutterStyle) => (
 		const classes = classNames(
```
[the remark this answers is not in the document]
Comment (zepfan) [reply src/elements/Grid/Grid.js]: Is there a specific reason why you prefer to do it this way over the other? Seems like it accomplishes the same thing but with more code

[the remark this answers is not in the document]
Comment (jfurfaro) [reply src/elements/Grid/Grid.js]: The classnames API was designed to handle this specifically, so if we stick with the API then we have a way to consistently check and apply syntax rules

[the remark this answers is not in the document]
Comment (jfurfaro) [reply src/elements/Grid/Grid.js]: and if you wanna be technical, this is an identical amount of characters, but the longer the list of boolean checks, the more characters it saves ;-)

[the remark this answers is not in the document]
Comment (zepfan) [reply src/elements/Grid/Grid.js]: It's really just the extra level of indentation that pisses me off lol

**Pull request merged**